src/components/Footer.jsx

```diff
@@ -1,34 +1,16 @@
-import { HiLink } from "react-icons/hi";
-import { IoMdMail } from "react-icons/io";
-import { PiTiktokLogo } from "react-icons/pi";
-import { FaDiscord } from "react-icons/fa";
-import { AiFillInstagram } from "react-icons/ai";
+import { footer } from "@/data/Footer.js";
```

Comment:
```suggestion
import { FOOTER } from "@/data/Footer.js";
```

for data files, names should be all caps